.gitignore

```diff
@@ -2,6 +2,7 @@
 *.log
 test.ponyfill.js
 test.polyfill.js
+dom.d.ts
```

Comment:
not sure why we would ignore this file. will it change every time `ts-graft` runs?